spec/factories/filesystem.rb

```diff
@@ -36,4 +36,189 @@
   factory :filesystem_txt_file, :parent => :filesystem do
     name "periodical/utilities/blue_screen_description.txt"
   end
+
+  #######################################
+  ##### Variations of nice format #######
```

Comment:
@matthewd maybe pointing me to how it should work? Or what do you mean? :-)

It's done like this on other places.